src/enclave/enclave_mem.c

```diff
@@ -17,6 +16,8 @@
 #include "enclave/ticketlock.h"
 #include "enclave/sgxlkl_config.h"
 
+static struct sysinfo;
```

Comment:
This looks like a type name without a variable name.  What does this actually do?

Reply:
This just declares to the compiler `struct sysinfo` is a type, so it doesn't complain anything like `struct sysinfo*`.

Reply:
Please remove the `static` here (it doesn't mean anything and I suspect it's not actually valid C).  The entire line is unnecessary though, because it's a forward declaration that comes after the real declaration.  Fields of this struct are used after this line with no other `sysinfo` declaration after this point.  This line should be removed.

Actually, after your most recent change, `sysinfo` isn't used in this file, so the forward declaration isn't needed.